src/screens/SettingsScreen.js

```diff
@@ -77,6 +78,19 @@ class SettingsScreen extends React.Component<Props> {
               value={store.get('playerID')}
             />
           </View>
+          <View style={styles.userIdFlex}>
+            <Text style={styles.saveText}>Off     </Text>
+            <Slider
+            style={{ width: 100, marginTop:10 }}
+            step={1}
+            value={store.get('testBool')}
+            minimumValue={0}
+            maximumValue={1}
+            minimumTrackTintColor='#FFFFFF'
+            onValueChange={val => store.set('testBool')(val)}
```

Comment:
`testBool` is only a number because we want the slider to work. in other words, semantically it should be a boolean, but for this specific UI we need a number.

from a code quality perspective, it makes more sense to store `testBool` as a boolean flag and do a conversion from number to bool in the `onValueChange` callback.